Tests.Integration.Presentation.Web/Contract/V2/ItContractsApiV2Test.cs

```diff
@@ -734,7 +736,7 @@ public async Task Can_PATCH_With_GeneralData()
             AssertGeneralDataSection(generalDataWriteRequestDto, contractType, contractTemplateType, agreementElements, freshDTO);
 
             //Act - new values
-            (contractType, contractTemplateType, agreementElements, generalDataWriteRequestDto) = await CreateGeneralDataRequestDTO(organization, false, true, false);
+            (contractType, contractTemplateType, agreementElements, generalDataWriteRequestDto) = await CreateGeneralDataRequestDTO(organization, false, true, false, false);
```

Comment:
remember to update the AssertGeneralData*** to check if what you set was actually written